website_and_docs/content/documentation/webdriver/elements/information.en.md

```diff
@@ -398,11 +402,43 @@ val text = driver.findElement(By.cssSelector("h1")).getText()
 
 
 
-## Attributes and Properties
+## Fetching Attributes or Properties
 
-### Attribute
+Fetches the run time value associated with a 
```

Comment:
Can you add these changes in other translated pages !

Reply:
in other pages, i am not familiar with the language, and some text is there in other language pages.

Reply:
The current process is to make the changes in English in all pages and someone else will be able to follow up and do the translations.

Reply:
done.